viewing_party/party.py

```diff
@@ -1,23 +1,223 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    # wave part 1
+    if title is None or genre is None or rating is None:
+        return None
+    
+    else:
+        dict = {}
+        dict["title"] = title
+        dict["genre"] = genre
+        dict["rating"] = rating
+        return dict

```

Comment:
Great logic here! A few small notes:

- The else is unneeded because the if will return early if triggered.
- Consider using a more descriptive variable name than `dict`.
- Consider making the dictionary all at once:
```
new_movie = {
        "title": title,
        "genre": genre,
        "rating": rating
}
```